phpstan-baseline.neon

```diff
@@ -2,7 +2,7 @@ parameters:
 	ignoreErrors:
 		-
 			message: "#^Function phpDocumentor\\\\Guides\\\\DependencyInjection\\\\template not found\\.$#"
-			count: 7
+			count: 8
```

Comment:
I assume you will get a conflict when the global directive is merged (as it also changes this line).

Reply:
yeah, taking care of it